.github/workflows/security_owasp.yml

```diff
@@ -21,36 +27,30 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       security-events: write
-    env:
-      # parent_directory: ${{ github.event.repository.name == '' && '.' || github.event.repository.name}}
-      serif_directory:  "${{ github.event.repository.name == '' && '.' || github.event.repository.name}}"
```

Comment:
this was never used as was a typo anyway - should have been `sarif_directory`

Reply:
oops!